tarodb/asset_minting_test.go

```diff
@@ -590,6 +590,54 @@ func TestCommitBatchChainActions(t *testing.T) {
 	}
 }
 
+// TestHeightHint checks that we can retrive the genesis tx height hint.
+func TestHeightHint(t *testing.T) {
+	ctx := context.Background()
+	const (
+		numSeedlings        = 5
+		numBatches          = 5
+		heightHint   uint32 = 123
+	)
+	assetStore, _, _ := newAssetStore(t)
+
+	// First, we'll create a new batch, then add some sample seedlings, and
+	// then those seedlings as assets.
+
+	// Add a few assets, and keep the last one around for the remainder of
+	// the test. We do this to ensure there are more than one batch in the
+	// database, such that we can ensure we are getting the correct one.
+	var (
+		batchKey   *btcec.PublicKey
+		genesisPkt *tarogarden.FundedPsbt
+		scriptRoot []byte
+	)
+
+	for i := 0; i < numBatches; i++ {
+		batchKey, genesisPkt, scriptRoot, _ = addRandAssets(
+			t, ctx, assetStore, numSeedlings,
+		)
+		genesisPkt.Pkt.Inputs[0].FinalScriptSig = []byte{}
+	}
+
```

Comment:
Query here to confirm that the batch tx is nil and the left join works when this phase hasn't been reached yet?